### ISSUE TITLE
Nginx 500 error does not show the proper error due to missing error template

### ISSUE THREAD

**Issue by joeribekker** [open-zaak/open-zaak#991]
**General information**
Open Zaak version: 1.4.0

**Describe the bug**
The [Ansible collection Nginx template](https://github.com/open-zaak/ansible-collection/blob/main/roles/open_zaak_nginx/templates/openzaak.conf.j2#L109):

```
{
    # ...

    error_page  404              /404.html;
    error_page  413              /413.json;
    error_page  500 502 503 504  /500.json;
}
```

The `/500.json;` doesn't resolve to a valid path. It seems Open Zaak doesn't have this file either.

The Nginx error:
```
2021/06/21 09:46:00 [error] 1302380#0: *744484 open() "/usr/share/nginx/html/500.json" failed (2: No such file or directory)
```

**To Reproduce**
1. Some method to trigger an Nginx 500 error (like upstream ends prematurely).

**Expected behavior**
A proper JSON error should be shown. Preferably with the Nginx error.

**Comment by sergei-maertens** [comment]
This definitely used to work in the reference impementations, we'll have to check which routes to set up for that again.

**Comment by sergei-maertens** [comment]
#1130 duplicated this and was fixed in #1144
